nodebuilder/node/metrics.go

```diff
@@ -42,9 +51,22 @@ func WithMetrics() error {
 		}
 
 		observer.ObserveFloat64(totalNodeRunTime, time.Since(timeStarted).Seconds())
+
+		// Observe build info with labels
+		labels := metric.WithAttributes(
+			attribute.String("buildTime", buildTime),
+			attribute.String("lastCommit", lastCommit),
+			attribute.String("semanticVersion", semanticVersion),
+			attribute.String("systemVersion", systemVersion),
+			attribute.String("golangVersion", golangVersion),
+		)
+
+		observer.ObserveFloat64(buildInfoGauge, 1, labels)
```

Comment:
why 1 is here? 😅

Reply:
this is because we are using `meter.Float64ObservableGauge`, and we are using attributes to add all the values that we need (strings), we need to set the value to the metric

for example, the content of one metric looks like this:
```
build_info{
  buildTime="Mon Jul 17 12:41:54 CEST 2023", golangVersion="go1.20.4", 
  instance="12D3KooWLfRoZnjLd1NsyKgS175iEqe1Y4okfhr4sDbpd1MsC1GB",
  job="Celestia-Full/semver-v0.0.2-24-gcde05d0-dev", 
  lastCommit="cde05d0a586c762c3de81f6b35c989c051f603a1", 
  semanticVersion="v0.0.2-24-gcde05d0-dev", 
  systemVersion="amd64/linux"
}
```

but we have to set some value to the metric itself:

![Screenshot 2023-07-17 at 13 25 32](https://github.com/celestiaorg/celestia-node/assets/32740567/60d18094-dc8d-49d7-bd20-50a29a7f1dc4)

Reply:
Wait. Can't we use another another type of metric? I am generally confused as it means all nodes in the same panel will have an gauge equal to 1, right?

Reply:
> Wait. Can't we use another another type of metric? I am generally confused as it means all nodes in the same panel will have an gauge equal to 1, right?

The value of `1` is used to indicate that the metric is **present/active**, it's like this [article](https://www.robustperception.io/exposing-the-software-version-to-prometheus/), bc the goal is not to provide a quantitative measurement but rather to expose the values using the attributes


cc: @walldiss

Reply:
Gauge is the correct choice here, you are right. It is there just to submit data with attributes, metric value is not important.

Reply:
Sometimes, I feel like the amount of dirty-hacks is just pilling up 😸 
All good for me then. Resolving this convo. Thanks for clarifications

Reply:
Ah. I don't have rights to resolve the convo. 
@walldiss please consider this as resolved 🙏